### ISSUE TITLE
Wants to write into some wrong /public/ directory.

### ISSUE THREAD

**Issue by nextgenthemes** [Elhebert/laravel-mix-sri#12]
```js
const mix = require( 'laravel-mix' );
require( 'laravel-mix-sri' );

// Sass configuration.
const sassConfig = {
	outputStyle: 'expanded',
	indentType: 'tab',
	indentWidth: 1,
};

//mix.sourceMaps();
//mix.setPublicPath( 'dist' );
//mix.version();
mix.js( 'src/arve-pro.js', 'dist' );
//mix.sass( 'src/arve-pro.scss', 'dist', sassConfig );
mix.generateIntegrityHash();

if ( process.env.sync ) {

	mix.browserSync( {
		proxy: 'symbiosistheme.test',
		files: [
			'dist/**/*',
			'**/*.php',
		],
	} );
}
```

```
Error: ENOENT: no such file or directory, open '/uncrypted/websites/s.com/web/app/plugins/arve-pro/public/dist/arve-pro.js'
```

should be just `dist` not `public/dist` tried reducing it to a absolute minimum config but did not help. I ran the production command.

**Comment by Elhebert** [comment]
There's a PR to make use of the `Config.publichPath` which should fix this #13

**Comment by nextgenthemes** [comment]
I specifically commented out the `//mix.setPublicPath( 'dist' )` did not work with or without. That PR may fix both cases because a default or something just wanted to mention it.

**Comment by Elhebert** [comment]
The issue should be fixed by #13
